tests/pipeline/test_engine.py

```diff
@@ -1207,6 +1221,15 @@ def test_dollar_volume(self):
         expected_5 = rolling_mean((self.raw_data ** 2) * 2, window=5)[5:]
         assert_frame_equal(results['dv5'].unstack(), expected_5)
 
+        expected_1_nan = (self.raw_data_with_nans[5:]
+                          * self.raw_data[5:] * 2).fillna(0)
```

Comment:
You can still just do `(self.raw_data_with_nans ** 2) * 2` here. Multiplying by `self.raw_data` will just fill with NaNs in the same places.